### PR TITLE
add urls to local and website

### PR DESCRIPTION
**Pull request by gnmearacaun**
Handy url to website and url when running locally

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # LunarVim.org
 
-Website for LunarVim
+[Website](https://www.lunarvim.org/) for [LunarVim](https://github.com/ChristianChiarulli/LunarVim)
+
+j
```
Comment (abzcoding) [inline README.md:5]: can you remove the `j` please 🥺

**Comment by Binx-Codes** [comment]
@gnmearacaun I don't think this link to the `Website` text is required. As the user can only see this through the lunarvim.org website itself

image

File: README.md (hunk 1 above)
Comment (gnmearacaun) [reply README.md]: Done ;)

**Comment by abzcoding** [comment]
> @gnmearacaun I don't think this link to the `Website` text is required. As the user can only see this through the lunarvim.org website itself
> 
> image

@gnmearacaun i do agree with binx here

**Comment by gnmearacaun** [comment]
> @gnmearacaun I don't think this link to the `Website` text is required. As the user can only see this through the lunarvim.org website itself
> 
> image

I personally find it helpful to have a link to the site in case the repo is landed on from somewhere other than the site, if I follow your thinking.

**Comment by Binx-Codes** [comment]
But how would it be landed on from another site, when there is only one lunarvim.org website? 🤔 Because since the user is already on the lunarvim.org site, there is no use to add the link to it again as they're in there already

**Comment by gnmearacaun** [comment]
The user can only see the repo through the lunarvim.org website? Sorry, it's not clear to me.

**Comment by gnmearacaun** [comment]
Oh, but I was thinking this was the readme to the repo, not visible on the website.

**Comment by gnmearacaun** [comment]
@Binx-Codes  @abzcoding  I do not see this page on the website. Do you have a link?

**Comment by abzcoding** [comment]
`[Website](https://www.lunarvim.org/)` this part is unnecessary :smile: because we are already reading it on `www.lunarvim.org`

**Comment by gnmearacaun** [comment]
> `[Website](https://www.lunarvim.org/)` this part is unnecessary smile because we are already reading it on `www.lunarvim.org`

**Comment by gnmearacaun** [comment]
> `[Website](https://www.lunarvim.org/)` this part is unnecessary smile because we are already reading it on `www.lunarvim.org`

Where, lol. Show me, I need to see a link to understand!

**Comment by abzcoding** [comment]
> > `[Website](https://www.lunarvim.org/)` this part is unnecessary smile because we are already reading it on `www.lunarvim.org`
> 
> Where, lol. Show me, I need to see a link to understand!
i mean we can make the title a link :)

image

**Comment by gnmearacaun** [comment]
If I follow, you saying that the links are redundant this page is viewed through the website. My understanding is that this is the readme for the repo and not visible on lunarvim.org. That's why I'm asking you to see link. Seeing is believing.

**Comment by gnmearacaun** [comment]
If I follow, you saying that the links are redundant this page is viewed through the website. My understanding is that this is the readme for the repo and not visible on lunarvim.org. Is this page on the website?

**Comment by abzcoding** [comment]
> If I follow, you saying that the links are redundant this page is viewed through the website. My understanding is that this is the readme for the repo and not visible on lunarvim.org. Is this page on the website?

at first yes I meant that :sweat_smile: , but after a while I understood my error,
right now I'm saying that as shown in the picture, we can do this
```md
# [LunarVim.org](https://www.lunarvim.org/)

Website for [LunarVim](https://github.com/ChristianChiarulli/LunarVim)
```

**Comment by gnmearacaun** [comment]
No problem. For a few minutes there I wasn't sure if I was walking against the wind or peeing into it :laughing:

**Comment by abzcoding** [comment]
> No problem. For a few minutes there I wasn't sure if I was walking against the wind or peeing into it laughing

my bad :sweat_smile:

**Comment by gnmearacaun** [comment]
No worries, we're not here to be perfect, we're here to be real

**Pull request merged**